recipes/gromacs/2020/build.sh

```diff
@@ -115,11 +115,11 @@ function _gromacs_bin_dir() {
         "${PREFIX}/bin.AVX_512/identifyavx512fmaunits" | grep -q '2' && \
         arch='AVX_512'
     ;;
-    *avx2*)
+    *\ avx2\ | *avx_2*)
```

Comment:
should the first expression be `*\ avx2\ *`? (I ask as I don't know if the search has to match the whole string, or just a subset of the string)

Reply:
Yes, it does need a trailing `*`.

Reply:
Noted, will fix once I see how the mac CI result is